tests/integration/test_gardenlinux.py

```diff
@@ -264,10 +280,10 @@ def test_growpart(client, openstack, openstack_flavor):
     sgb = int(lines[1].strip()[:-1])
     assert sgb == expected_disk_size, f"partition size expected to be ~{expected_disk_size} GB but is {sgb}"
 
-def test_docker(client):
```

Comment:
Why is the docker test failing on kvm?

Reply:
I tested the kvm test with the kvm image, the kvm image is build without the gardener feature. The gardener feature installs docker. 
Today I tried to test the gcp-gardener image with kvm, docker is installed, but the image has only about 8MB of space left, the docker test tries to pull an image that is >25MB, the test fails with no space left on device.
I guess we have two options,

1. Not running the test in kvm or
2. check if the feature gardener was used for building the image and resize the image for the test.

Reply:
Option two sounds good. I believe we should have these tests for the Gardener feature already run in KVM. Right now, the Gardener feature is the most important one for us as we are still primarily releasing Garden Linux for use with Gardener.

Reply:
changed the test and the kvm environment to make the test work in kvm as suggested in option 2.